config.js

```diff
@@ -30,5 +30,24 @@ var config = {
       destination : "", // Destination of your trip. Human readable address.
       name : "work", // Name of your destination ex: "work"
       reload_interval : 5 // Number of minutes the information is refreshed
+    },
+    fitbit: {
+        "timeout": 10000,
+        "creds": {
+            "clientID": "CLIENTID",
+            "clientSecret": "SECRET"
+        },
+        "uris": {
+            "authorizationUri": "https://www.fitbit.com",
+            "authorizationPath": "/oauth2/authorize",
+            "tokenUri": "https://api.fitbit.com",
+            "tokenPath": "/oauth2/token"
+        },
```

Comment:
Agreed - main config should be as simple as possible. I've now modified it so that the only fitbit config item in the main config.js is to enable or disable fitbit integration. This setting then determines whether the API is actually called or not, and whether or not the fitbit stats show in the UI or not, as well as voice command registration. The fitbit API URIs, and ID/secret are now setup in the fitbit-service.js and a comment in the config.js file directs fitbit users there. Much cleaner!